nginx-aspnet-mysql/backend/Program.cs

```diff
@@ -1,84 +1,62 @@
-                        context.Response.StatusCode = 500;
-                        return Task.CompletedTask;
-                    }
-                    connection.Close();
-                    
-                    context.Response.StatusCode = 200;
-                    context.Response.WriteAsJsonAsync(titles);
+app.MapGet("/", (MySqlConnection connection) => {
+    var titles = new List<string>();
 
-                    return Task.CompletedTask;
-                });
-            });
-            Prepare(connectionString);
+    try
+    {
+        Console.WriteLine("Connecting to MySQL...");
+        connection.Open();
 
-        }).Build().Run();
+        string sql = "SELECT title FROM blog";
+        using var cmd = new MySqlCommand(sql, connection);
+        using MySqlDataReader reader = cmd.ExecuteReader();
 
-        private static void Prepare(string connectionString)
+        while (reader.Read())
         {
-            using MySqlConnection connection = new MySqlConnection(connectionString);
-
-            connection.Open();
-            using var transation = connection.BeginTransaction();
-
-            using MySqlCommand cmd1 = new MySqlCommand("DROP TABLE IF EXISTS blog", connection, transation);
-            cmd1.ExecuteNonQuery();
-
-            using MySqlCommand cmd2 = new MySqlCommand("CREATE TABLE IF NOT EXISTS blog (id int NOT NULL AUTO_INCREMENT, title varchar(255), PRIMARY KEY (id))", connection, transation);
-            cmd2.ExecuteNonQuery();
-            
-            for (int i = 0; i < 5; i++)
-            {
-                using MySqlCommand insertCommand = new MySqlCommand( $"INSERT INTO blog (title) VALUES ('Blog post #{i}');", connection, transation);
-                insertCommand.ExecuteNonQuery();
-	        }
-            transation.Commit();
-            connection.Close();
+            titles.Add(reader.GetString(0));
         }
+        reader.Close();
+    }
+    catch (Exception ex)
+    {
+        return Results.Problem(detail: ex.ToString());
+    }
+    connection.Close();
+    
+    return Results.Ok(titles);
+});
+Prepare(connectionString);
+
+app.Run();
+
+void Prepare(string connectionString)
+{
+    using MySqlConnection connection = new MySqlConnection(connectionString);
+
+    connection.Open();
+    using var transation = connection.BeginTransaction();
+
+    using MySqlCommand cmd1 = new MySqlCommand("DROP TABLE IF EXISTS blog", connection, transation);
+    cmd1.ExecuteNonQuery();
+
+    using MySqlCommand cmd2 = new MySqlCommand("CREATE TABLE IF NOT EXISTS blog (id int NOT NULL AUTO_INCREMENT, title varchar(255), PRIMARY KEY (id))", connection, transation);
+    cmd2.ExecuteNonQuery();
+    
+    for (int i = 0; i < 5; i++)
+    {
+        using MySqlCommand insertCommand = new MySqlCommand( $"INSERT INTO blog (title) VALUES ('Blog post #{i}');", connection, transation);
+        insertCommand.ExecuteNonQuery();
+    }
+    transation.Commit();
+    connection.Close();
 }
```

Comment:
Nit: add new line at the end of the file

Reply:
Done.